### PR TITLE
Update First-Installation.md

### PR DESCRIPTION
**Pull request by GregariousJB**
**Hardware section**

* Couldn't tell that image was a link to a video, so I Photoshopped in a translucent "Play" icon on top of it, and also made it a link in the text just above it. Shrank it down so it's not so huge on the page.
* Brought that Canakit link up to the top and also updated it (was showing a Pi 3 kit). I pointed it to the main Canakit brand page on Amazon so it shouldn't need to be updated with each new Pi model.
* Split into "Required" and "Optional" sections for clarity. 
* Added several unbreakable image links to items that some people may be unfamiliar with.
* Attempted future-proofing this instruction so it doesn't need to be changed after each new RPI that gets released. Somewhat failed. Think it's probably best to reference the last few RPI models (3 and 4), otherwise this instruction will get very thin and could make things worse.
* Simplified some of the lines to coincide with [RPI's official hardware page](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/1), like the "Power supply" section.
* I removed references to RCA. I think it's cool that RCA is supported but instructions can be found easily outside of this page for the advanced users that will be using it. It's probably best to keep things within this century for the sake of simplicity (HDMI).

### CODE REVIEW

**Comment by dankcushions** [comment]
I think you should lose the articles from the bullet points. eg
> A MicroSD Card
becomes
> MicroSD Card

It just reads strangely to me, and is inconsistent with the rest of the docs. 

I know it was there previously, but I think we should replace the canakit recommendation with a bundle that includes the official PSU, such as https://www.amazon.co.uk/dp/B07V9Y1SR4/ref=cm_sw_r_cp_api_fabc_N8BGW0RC4JA6GGCP1ZHD - i am not sure about pi4, but canakit’s bundled pi3 psu was infamous for causing undervoltage issues; the official psu is always reliable.

**Comment by GregariousJB** [comment]
> I think you should lose the articles from the bullet points

Done.

As for the power supply, [the Overclocking page seems to encourage the Canakit's PSU](https://retropie.org.uk/docs/Overclocking/#power-supply), and I don't think it's been updated for the Pi 4 yet. 🤷‍♂️

**Comment by dankcushions** [comment]
That overclocking page is full of bad information - like i said, [on my hit list](https://github.com/RetroPie/RetroPie-Docs/pull/55#issuecomment-772348515) :)

but yeah - it's [infamous](https://www.google.com/search?q=canakit+undervolt+retropie+site:retropie.org.uk) for it.

**Comment by GregariousJB** [comment]
Fair enough. I trust you know more about it than I do. 

That being said, I don't want to make a change to recommend something I know nothing about, so if you want to make a PR about it after this one that's up to you.

**Comment by cmitu** [comment]
Looks better, my only notes are
* The Pi Zero/1 have miniHDMI, you might want to include that in the list. While the Pi 1 model isn't manufactured, the Zero is still active, though they're mainly used for portable kits.
* Leave the RCA TV connection on. For a retro-gaming kit, I think it's still relevant and there are users that use this setup with an old CRT TV.
* The formula _Pi 3+ models_ for WiFi support is a bit confusing, since there is actually a Pi 3(B)+ model. Maybe change it to _PI 3 and later models_.

**Comment by GregariousJB** [comment]
Cool. I think I got everything.

I'm tempted to add `(included with Canakit)` to some of these, like the MicroSD card reader and the case, but I don't want to seem like a Canakit shill. 

I do think having recommendations is useful, like a link to the awesome FLIR case, but where is the line crossed? Should we also have controller recommendations, cable recommendations, etc? It'll make it easier for brand new people at the expense of looking busier and potentially more difficult to navigate. Is there a "Recommendations" section elsewhere that can be linked to? [This page on the website](https://retropie.org.uk/about/building/) isn't very thorough, unfortunately. Maybe this is a non-issue that I'm thinking too much about.

File: docs/First-Installation.md [hunk 1]
```diff
@@ -9,26 +9,39 @@ This page is for people just getting started on RetroPie. The easiest way to ins
...
+ * MicroSD Card ([list of compatible SD cards](http://elinux.org/RPi_SD_cards))
+ * Screen (TV, computer monitor, projector, etc) - anything with HDMI
+ * Video cable (HDMI is best)
+    * Pi 4 will need a [Micro HDMI to HDMI](https://user-images.githubusercontent.com/540857/107463741-c3a47480-6b2c-11eb-8402-7313a490e234.jpg) cable
+    * Pi 1, 2, and 3 will need a full-size [HDMI](https://user-images.githubusercontent.com/540857/107463845-ff3f3e80-6b2c-11eb-8031-3d0ed5563861.jpg) cable
+    * Pi Zero will need a [Mini HDMI to HDMI](https://user-images.githubusercontent.com/540857/107686191-e3de4b80-6c72-11eb-8162-07a82afaac7b.jpg) cable
+    * [4-Pole RCA to 3.5mm](https://user-images.githubusercontent.com/540857/107688722-f3ab5f00-6c75-11eb-9837-ffe7f385b1c5.jpg) is also an option for older screens
+ * Power supply
+    * At least 3.0 amps for Raspberry Pi 4
+    * At least 2.5 amps for Raspberry Pi 3
```
Comment (dankcushions) [inline docs/First-Installation.md:34]: Please re-add the voltage requirements, and include the requirements for pi0-2 - see https://www.raspberrypi.org/documentation/faqs/#pi-power-specs

File: docs/First-Installation.md [hunk 2]
```diff
@@ -9,26 +9,39 @@ This page is for people just getting started on RetroPie. The easiest way to ins
...
+ * Game controller of your choice 
+    * Can be USB-wired, wireless (with a dongle), or Bluetooth (with or without a dongle. Pi 3 and later models have built-in bluetooth and won't need a dongle)
+    * The [Control Block](http://blog.petrockblock.com/2014/12/29/controlblock-power-switch-and-io-for-the-raspberry-pi/) can use original SNES controllers
+
+### Optional
+
+ * Raspberry Pi case - **highly recommended** for proper cooling and better performance
```
Comment (dankcushions) [inline docs/First-Installation.md:41]: better performance? i don't think i'd go that far. some restrict cooling, even. i think i would be more ambiguous and just say "(but recommended)".

File: docs/First-Installation.md [hunk 3]
```diff
@@ -9,26 +9,39 @@ This page is for people just getting started on RetroPie. The easiest way to ins
 
 This guide will give you the very basics to get you up and running from a blank SD card to first boot into EmulationStation.
 
-If you hate reading then see this video. Otherwise read on! 
+If you hate reading then see [this video](https://www.youtube.com/watch?v=E1sbnPZ_A8w). Otherwise read on!
```
Comment (dankcushions) [inline docs/First-Installation.md:13]: no need for a link if it's embedded below

**Comment by dankcushions** [comment]
> I'm tempted to add (included with Canakit) to some of these, like the MicroSD card reader and the case, but I don't want to seem like a Canakit shill

i will be replacing the previous canakit recommendation as discussed :)

> I do think having recommendations is useful, like a link to the awesome FLIR case, but where is the line crossed? Should we also have controller recommendations, cable recommendations, etc? It'll make it easier for brand new people at the expense of looking busier and potentially more difficult to navigate. Is there a "Recommendations" section elsewhere that can be linked to? This page on the website isn't very thorough, unfortunately. Maybe this is a non-issue that I'm thinking too much about.

yeah, i don't think we should include recommendations for that kind of thing. a lot of it is contextual, anyway. eg, no point in spending a premium on a flirc if you're not planning on overclocking, and i'm not sure i want to put my hat in the ring for any particular cable manufacturer, never mind controllers!

**Comment by GregariousJB** [comment]
Done. I also found that Raspberry Pi power page and decided it's probably best to just link to it. That'll future-proof the instruction a bit more as an added bonus. 

Thought of adding `for proper cooling and required for [Overclocking](Overclocking)` to the case recommendation, but I think I'll just leave it alone.

**Comment by GregariousJB** [comment]
image

Um. What happened to the list formatting?

**Comment by cmitu** [comment]
Too many spaces. Should be fixed now.

**Comment by GregariousJB** [comment]
Just hosted this site locally on my PC to take a closer look. 

Apparently 4 spaces works fine with Github's markdown, and with Material by default, but not when [mdx_truly_sane_lists](https://github.com/radude/mdx_truly_sane_lists) is included, which defaults to 2 spaces. 

Is this extension configured somewhere other than the mkdocs.yml? Unless I'm missing something, you might want to consider setting `nested_indent` from 2 (default) to 4 (Github markdown and Material default).

It might break a few lists in a few pages, but it would mean nested lists are uniform across markdown and this questionable extension.

**Comment by cmitu** [comment]
I'll take a look to see if any configuration might accommodate both MD implementations (Github and python-md+mdx_truly_sane_lists).

**Pull request merged**